python/cudf/cudf/core/column/numerical.py

```diff
@@ -396,9 +396,10 @@ def can_cast_safely(self, to_dtype):
             else:
                 from cudf import Series
```

Comment:
Don't do the import in here, just use `cudf.Series` within the function. Doing local imports like this within a function is actually surprisingly expensive, even if the import is cached.

Reply:
I think the problem with this was that it resulted in a circular import scenario. Looking into solutions.

Reply:
IIRC we already have an `import cudf` at the top of this file, where inside this function we can just use `cudf.Series` instead of `Series`.